### DISCUSSION TITLE
Trying to use Firefly+FIDI docker compose - .db.env error

### DISCUSSION THREAD

**Discussion by AngryEyebrows** [firefly-iii/firefly-iii#6454] [Firefly III]
HI, I am trying to use the docker compose for the combined firefly + fidi + mariadb installation. 

I get an error when trying trying to bring it up with 'docker-compose -f docker-compose.yml up -d'

 ERROR: Couldn't find env file: /etc/docker/firefly/.db.env

There wasn't a reference in the guide for an example .db.env I can use...Is there anywhere I can get one or should it be creating this as part of the install?

**Comment by JC5** [comment]
Which composer file are you using exactly?

**Reply by AngryEyebrows** [reply]
Hi JC5, I am using the one at the bottom of the page here - https://docs.firefly-iii.org/data-importer/install/docker/

**Reply by JC5** [reply]
You'll have to write your own .env file using the example variables from the maria db docker page:

https://hub.docker.com/_/mariadb

Example:

```
MARIADB_RANDOM_ROOT_PASSWORD=yes
MYSQL_USER=firefly
MYSQL_PASSWORD=supersecret
```
